cmd/snap/cmd_pack.go

```diff
@@ -36,6 +36,7 @@ import (
 
 type packCmd struct {
 	CheckSkeleton bool   `long:"check-skeleton"`
+	AppendVerity  bool   `long:"append-dm-verity-data"`
```

Comment:
should we s/dm-verity/integrity/ in this file?

Reply:
Fixed. Should I also s/AppendVerity/AppendIntegrity/? I kinda left it on purpose because since it is only about dm-verity at the moment made more sense.